packages/sdk/src/chains/EVM/types/index.ts

```diff
@@ -0,0 +1,127 @@
+import { providers, ethers } from 'ethers';
+import { FeeDataResult } from 'types';
+import { Bridge, ERC20, ERC721MinterBurnerPauser } from '@buildwithsygma/sygma-contracts';
+
+export type TokenConfig = {
+  /** The token type (ERC20 or ERC721) */
+  type: 'erc20' | 'erc721';
+  /** The address of the token contract. */
+  address: string;
+  /** The optional name of the token. */
+  name?: string;
+  /** The optional symbol of the token. */
+  symbol?: string;
+  /** The optional URI of the token image. */
+  imageUri?: string;
+  /** The resource identifier of the token. */
+  resourceId: string;
+  /** An optional flag indicating if the token is a native wrapped token. */
+  isNativeWrappedToken?: boolean;
+  /** The optional number of decimals for the token. */
+  decimals?: number;
+  /** An optional flag indicating if the token requires double approval (like USDT). */
+  isDoubleApproval?: boolean;
+  /** The fee settings for the token. */
+  feeSettings: {
+    /** The type of fee, either 'basic', 'feeOracle', or 'none'. */
+    type: FeeType;
+    /** The address of fee handler contract. */
+    address: string;
+  };
+};
+/**
+ *  Fee startegies of the Bridge
+ *  Where "basic" stands for fixed fee and "feeOracle" is for dynamic
+ */
+export type FeeType = 'basic' | 'feeOracle' | 'none';
+/**
+ *  The config of the bridge
+ */
+export type ProcessTokenTranferBridgeConfigParamsType = {
```

Comment:
Can you please make some suggestion then and not just resolve it.

Reply:
This is bridge config param which part of the params for processDeposit method only. So it could be 

```suggestion
export type ProcessTokenTranferBridgeConfigParam = {
```

```suggestion
export BridgeConfigParam = {
```

Reply:
2. (BridgeConfigParam) one seems a lot better